src/components/utils/errors/errors.component.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx';
+import React from 'react';
+
+export interface IErrorsProps {
+  componentId: string;
+  errors: string[];
+}
+
+/**
+ * Reusable component showing `errors` as list of error messages for the component.
+ */
+export const Errors: React.FC<IErrorsProps> = ({componentId, errors}) => {
+  const className = clsx(`of-error-list`);
+  const listItemClassName = clsx(`of-error-list__item`);
+  const labelClassName = clsx(`of-error-list__message`);
+
+  return (
+    <ul className={className} aria-describedby={componentId}>
+      {errors?.map((error: string) => {
+        return (
+          <li key={error} className={listItemClassName}>
```

Comment:
the errors list is not guaranteed to be a unique list of errors, so your key is not guaranteed to be unique. I'd incorporate the array index here.

Reply:
Ss there an actual case where show the same error twice? Given that resolving an error might render a new error "field foo is required" with index 0 becomes "field foo does not match pattern bar" with index 0 which might lead to unwanted behavior?

Reply:
there shouldn't be but you can't rule it out. React still goes through the diffing algorithm if any of the props change, so if the error list itself is re-ordered, the `children` prop changes because the actual error text is different.

React itself advises to fall back to index if you cannot have a guaranteed stable identity/key for a given item in the iterable (while warning about the possible issues this has with mutating!).

Doing ```key={`${index}-${error}`}``` here would be guaranteed to be unique (though the index itself + child already does that)